### DISCUSSION TITLE
Can't choose proof type.

### DISCUSSION THREAD

**Discussion by fxzxmic** [community/community#67819] [GitHub Education]
### Hi everyone,

I can't choose the "other" proof type but the system ask me to do so.
There is no selection box.

**Comment by LiaoGuoYin** [comment]
Same as you, I can not find the selection box, But I found the others screenshot have one

Here is my page(No select box):
<img width="661" alt="image" src="https://github.com/community/community/assets/13686264/040157ae-0106-4456-86de-16fd13d74804">


After uploading my material, I got: 

`Your application cannot be reviewed until you fix the following: Please select proof type `

**Comment by PikeNote** [comment]
Same here, I am unable to upload my proof to complete/process my application here.

**Comment by vishalkrishna8** [comment]
Hey @fxzxmic try this, place your valid academic ID or other proof of current academic status in the frame, then click Take photo. Under "Proof Type", use the dropdown to select the type of proof you are providing.

**Reply by Z-astro** [reply]
When I chose to take a picture, GitHub even allowed me to use only the front camera, which was very inconvenient and idiot.

**Comment by fxzxmic** [comment]
@vishalkrishna8 Have you ever thought that I raised this question just because I don't have the "Proof Type" selection box there?

**Comment by CPettigrew33** [comment]
I am having the same problem. At any stage of submitting proof, a "proof type"  selection box never appears.

**Comment by shangji666** [comment]
same problem

**Comment by NickWang16** [comment]
I have the same problem，I tried with my iphone safari，PC chrome，edge，clean the cache，still no select box

**Reply by nihilisticzebra** [reply]
Yeah it seems like they make this process hard on purpose. Because those issues have remained unfixed for years now
